src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -76,6 +77,9 @@ public class TargetMethodFinderVisitor extends ModifierVisitor<Void> {
    */
   private Set<String> usedClass = new HashSet<>();
 
+  /** Enums that were actually used by the targets. These classes will be included in the input. */
+  private Set<String> usedEnum = new HashSet<>();
```

Comment:
Why do we need a separate set for used enums? I think it would work correctly if they were stored in the same set as used classes: that is, if they were added to `usedClass` instead of kept as a separate field.

(See my other comment below, as well, for why I think this is safe.)

Reply:
Professor, it's mostly because JavaParser does not consider an enum a class. So I thought we should also separate them.

Reply:
I disagree with JavaParser on this: under the hood, enums are compiled to classes, and they share the same namespace. Let's simplify our own code and just have one set of used class-like structures.

On a related note, we should settle on a term for "class-like structures" and use it throughout the code. `usedClass` isn't a great name anymore for that field - it's more like "used types" or "used concrete type declarations". Within the checker framework, we'd usually refer to such a thing as a `TypeElement`, following javac's internal convention.

Reply:
I'm a bit confused. If we consider an enum to be a class, then why `usedClass` is not a good name anymore?

Reply:
Because `usedClass` also contains interfaces (which are not classes, but are types) and enums (which are a special kind of class-like thing that javac will compile into a class). But this is a minor detail and shouldn't block this PR from being merged once we've applied the suggested refactoring.